tests/components/deconz/test_cover.py

```diff
@@ -95,11 +94,14 @@ async def test_cover(hass):
     gateway.api.event_handler(state_changed_event)
     await hass.async_block_till_done()
 
-    level_controllable_cover = hass.states.get("cover.level_controllable_cover")
-    assert level_controllable_cover.state == "closed"
+    assert hass.states.get("cover.level_controllable_cover").state == "closed"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)